.github/workflows/lint.yml

```diff
@@ -15,6 +15,9 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.7
+      - name: Install MMEngine
+        run: |
+          pip install git+https://github.com/open-mmlab/mmengine.git
```

Comment:
It still got lint error, which shows in line `from mmengine.fileio import dump`, no mmengine package in environment.